src/test/resources/features/02_interest.feature

```diff
@@ -73,21 +73,27 @@ Feature: Interest Journeys - Income Tax Submission
     Then the user should be redirected to the "Did your client get taxed interest from the UK?" page
     When the user selects the yes radio button
     And the user clicks the continue button
-    Then the user should be redirected to the "Add an account with taxed UK interest" page
+    Then the user should be redirected to the "Which account did your client get taxed UK interest from?" page
+    When the user selects the fifth radio button: add another new account
+    And the user clicks the continue button
     When the user selects the UK taxed interest account name field and enters a value of RBS
     And the user selects the UK taxed interest amount earned field and enters a value of 500
     And the user clicks the continue button
     Then the user should be redirected to the "Accounts with taxed UK interest" page
     When the user selects the yes radio button
     And the user clicks the continue button
-    Then the user should be redirected to the "Add an account with taxed UK interest" page
+    Then the user should be redirected to the "Which account did your client get taxed UK interest from?" page
+    When the user selects the fifth radio button: add another new account
+    And the user clicks the continue button
     When the user selects the UK taxed interest account name field and enters a value of Skipton
     And the user selects the UK taxed interest amount earned field and enters a value of 600
     And the user clicks the continue button
     Then the user should be redirected to the "Accounts with taxed UK interest" page
     When the user selects the yes radio button
     And the user clicks the continue button
-    Then the user should be redirected to the "Add an account with taxed UK interest" page
+    Then the user should be redirected to the "Which account did your client get taxed UK interest from?" page
+    When the user selects the fifth radio button: add another new account
```

Comment:
I don't think we should be using "Add another new account" here, it should reflect what the radio button says ("Add a new account")